route/journal-router.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const Journal = require('../model/journal.js');
```

Comment:
Require the http-errors module so you can make custom error statuses and messages. const createErrors = require('http-errors');